packages/esm-patient-orders-app/src/routes.json

```diff
@@ -35,6 +35,11 @@
         "fullWidth": false
       }
     },
+    {
```

Comment:
This is another thing I'm going to object to, since we already have an extension for this component. For now, I guess we can just change `slot` to `slots` in the existing definition, but we need to come up with something better long-term.

Reply:
Yes, ideally we should, but the original definition has an `order=0` which isn't the case with the `ward-patient` action menu.

Reply:
![image](https://github.com/user-attachments/assets/d8da6179-ace7-4120-9406-4c3e4b4a6c56)

The patient chart action menu button should remain at the top of the slot. I tried giving the patient chart action menu button an order of -1, but it didn't work, as the minimum value required is 0.

![image](https://github.com/user-attachments/assets/b8626bdb-6cb5-4d08-8615-d3da5be69668)

Thanks!